packages/web3-test-utils/src/accounts_utils.ts

```diff
@@ -0,0 +1,78 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import { Web3Context } from 'web3-core';
+import { getAccounts, sendTransaction } from 'web3-eth';
+import { create, decrypt, Wallet, privateKeyToAccount } from 'web3-eth-accounts';
+import { Address, Web3AccountProvider } from 'web3-types';
+import { DEFAULT_RETURN_FORMAT } from 'web3-utils';
+
+import { getTestClient, getTestProvider } from './get_env';
+
+export const createAccount = () => {
+    const newAccount = create();
+    return { address: newAccount.address, privateKey: newAccount.privateKey };
+}
+
+const fundGethAccount = async (fundeeAddress: Address, fundAmount: BigInt) => {
```

Comment:
Mostly logic for funding test accounts will remain same, we just can pass `from` address ( or coinbase in case of geth, or one of 10 pre funded ganache acct) or any provided by user. so can we omit diff implementation for diff nodes.